h/admin.py

```diff
@@ -186,6 +191,24 @@ def users_index(request):
 
     return {'username': username, 'user': user, 'user_meta': user_meta}
 
+@view.view_config(route_name='admin_users_delete',
```

Comment:
Convention is 2 blank lines between functions, as here, although it's 1 blank line between methods in classes, perversely.